### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by josephsavona**
Update the team page with additional folks working on programming model and learning/advocacy.

### CODE REVIEW

File: beta/src/content/community/team.md [hunk 1]
```diff
@@ -62,6 +86,10 @@ Current members of the React team are listed in alphabetical order below.
     Sathya hated the Dragon Book in school but somehow ended up working on compilers all his career. When he's not compiling React components, he's either drinking coffee or eating yet another Dosa.
 </TeamMember>
 
+<TeamMember name="Sean Keegan" permalink="sean-keegan" photo="/images/team/sean-keegan.jpg" github="seanryankeegan" twitter="DevRelSean" title="Developer Advocate at Meta">
+    After a first career as a math teacher, Sean remembered that one intro to comp sci class he had to take as a prereq and thought “that was kind of fun!” One coding bootcamp and several tech jobs later, Sean discovered developer advocacy and hasn’t looked back. Outside of work, Sean enjoys ultimate frisbee, video games, and researching (but rarely implementing) better ways to care for his houseplants.
```
Comment (poteto) [inline beta/src/content/community/team.md:90]:
```suggestion
    After a first career as a math teacher, Sean remembered that one intro to comp sci class he had to take as a prereq and thought “that was kind of fun!”. One coding bootcamp and several tech jobs later, Sean discovered developer advocacy and hasn’t looked back. Outside of work, Sean enjoys ultimate frisbee, video games, and researching (but rarely implementing) better ways to care for his houseplants.
```

File: beta/src/content/community/team.md [hunk 2]
```diff
@@ -18,10 +18,22 @@ Current members of the React team are listed in alphabetical order below.
     Andrew got started with web development by making sites with WordPress, and eventually tricked himself into doing JavaScript. His favorite pastime is karaoke. Andrew is either a Disney villain or a Disney princess, depending on the day.
 </TeamMember>
 
+<TeamMember name="Andrey Lunyov" permalink="andrey-lunyov" photo="/images/team/andrey-lunyov.jpg" github="alunyov" twitter="alunyov" title="Engineer at Meta">
+    I started my career as a designer and then gradually transitioned into web development. After joining the React Data team at Meta, I worked on adding an incremental JavaScript compiler to Relay, and then later on, I worked on removing the same compiler from Relay. Outside of work, I like to play music and engage in various sports.
```
Comment (poteto) [inline beta/src/content/community/team.md:22]: @alunyov Should we do 3rd person to be consistent?

**Review by poteto — APPROVED** [review]
🤍🤍🤍

File: beta/src/content/community/team.md (hunk 2 above)
Comment (gaearon) [reply beta/src/content/community/team.md]: yes we should

Comment (josephsavona) [reply beta/src/content/community/team.md]: edited

**Pull request merged**